docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -39,6 +39,79 @@ The Fairlearn package is most applicable to two kinds of harms:
   information are extended or withheld. Examples include varying accuracy in
   face recognition, document search, or product recommendation.
 
+Concept Glossary
+----------------------------
+
+The concepts outlined in this glossary are relevant to sociotechnical contexts. 
+
+Construct Validity
+^^^^^^^^^^^^^^^^^^
+
+We explore the concept of construct validity, as described in in Measurement and Fairness [#4]_ , and as it relates to sociotechnical contexts. This is intended to illuminate the concept to readers who are trying to build AI systems in sociotechnical contexts and may also be referenced in explorations that detail how to apply the concept to specific scenarios.
```

Comment:
I think we can make clear how this glossary relates to fairness in context! Maybe revise to something along the lines of:

"Readers working on measuring fairness are making assumptions and choices about what fairness means for their use case and context. This glossary is intended to provide readers with language and a framework for testing the validity of those assumptions in the sociotechnical context they are working in. Ideally, this glossary may be referenced by extended examples that detail how to apply the concepts to specific scenarios."

Reply:
Is *formal* an unobservable theoretical construct though @hildeweerts 
👀

Reply:
It might even be a highly contested one 👀